elasticsearch/Dockerfile

```diff
@@ -61,22 +36,7 @@ COPY utils/** /usr/local/bin/
 RUN ln -s /usr/local/bin/logging ${HOME}/logging
 
 RUN ${HOME}/install.sh
-RUN rm -r /tmp/lib
 
 WORKDIR ${HOME}
 USER 1000
 CMD ["sh", "/opt/app-root/src/run.sh"]
-
```

Comment:
I would add all this generated bits back in

Reply:
@jcantrill why?